### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by antsankov**
Fully synced a fresh server with ETC, wanted to make it clearer for users based on my mistake of using to little RAM

### CODE REVIEW

**Comment by jonringer** [comment]
cc @input-output-hk/mantis-devs

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ You can use generic launcher with appropriate parameter to connect with pre-conf
 Example:
   - `./bin/mantis-launcher etc` - for joining Ethereum Classic network
 
-Possible networks: `etc`, `eth`, `mordor`, `testnet-internal`
+Possible networks: `etc`, `eth`, `mordor`, `testnet-internal`. On a fresh Linux instance, it's recommended to use 8GB of memory to fully sync ETC, and have at least 100GB of disk space. Also `default-jdk` can be used if Java is not installed by default on the system.
```
Comment (lemastero) [inline README.md:30]: `default-jdk` can change in time & might vary between linux distros
What `java -version` did you used?

Comment (antsankov) [reply README.md:30]: I’ll check it out tomorrow on my server. Do you recommend I just change this section or remove it entirely and just leave hardware requirements?

**Comment by faraggi** [comment]
Oops. There seems to be conflicts with my new branch; https://github.com/input-output-hk/mantis/tree/docs/readme-update

I've been rehauling all mantis repo READMEs since last week

File: README.md (hunk 1 above)
Comment (dmitry-worker) [reply README.md:30]: @antsankov thanks for your contribution! 

Just 2 cents: I do not have `default-jdk` on `Ubuntu`.
But I'm not sure if JDK (not JRE) is required.

Mantis docker image is already shipped with JRE of course. 
We could probably build .deb or .rpm / .rpmi package that depends on JRE. WDYT?

**Comment by dzajkowski** [comment]
@antsankov first of all thank you for the contribution! greatly appreciated!
As @faraggi mentioned - we are working on a documentation refresh - we would like to include your changes as part of it and credit you. Does this seem fair?

**Comment by antsankov** [comment]
Yes that sounds great guys! I would recommend putting a minimum hardware requirements and recommended requirements would be very useful.

Closing PR now.

**Pull request closed without merging**